Makefile

```diff
@@ -0,0 +1,164 @@
+.ONESHELL:
+
+# Can be overridden to use a different directory name.
+ifeq ($(VENV),)
+  VENV := .venv
+endif
```

Comment:
Just for consideration, is it worth implementing a function to handle this check or set default behavior? 
something like:
```
define CheckOrSetDefault
  ifeq ($($1),)
    $1 := $2
  endif
endef
```
which could be called like
```
$(call CheckOrSetDefault, <VARIABLE>, <DEFAULT>)
```

Reply:
Good idea!